app/src/main/java/sdp/moneyrun/EndGameActivity.java

```diff
@@ -13,7 +16,7 @@
  */
```

Comment:
Even though the methods are straight forward and understandable at first sight, a general documentation 1-2 sentences could be useful in the future or in other classes